recipes/cmake/3.x.x/conanfile.py

```diff
@@ -137,7 +138,6 @@ def package_info(self):
         self.cpp_info.includedirs = []
 
         # TODO remove once conan v2 is the only support and recipes have been migrated
-        if Version(conan_version).major < 2:
-            bindir = os.path.join(self.package_folder, "bin")
-            self.output.info("Appending PATH environment variable: {}".format(bindir))
-            self.runenv_info.append_path("PATH", bindir)
+        bindir = os.path.join(self.package_folder, "bin")
+        self.output.info("Appending PATH environment variable: {}".format(bindir))
+        self.runenv_info.append_path("PATH", bindir)
```

Comment:
By default the bin folder is added to the virtualrunenv so I think this can be removed 🤔

Reply:
Removed